packages/ace-core/src/checker/checker-epub.js

```diff
@@ -37,79 +38,29 @@ function newViolation({ impact = 'serious', title, testDesc, resDesc, kbPath, kb
     .build();
 }
 
-function newMetadataAssertion(name, impact = 'serious') {
-  return newViolation({
-    impact,
-    title: `metadata-${name.toLowerCase().replace(':', '-')}`,
-    testDesc: `Ensures a '${name}' metadata is present`,
-    resDesc: `Add a '${name}' metadata property to the Package Document`,
-    kbPath: 'docs/metadata/schema-org.html',
-    kbTitle: 'Schema.org Accessibility Metadata',
-  });
-}
-
-function checkMetadata(assertions, epub) {
-  // Required metadata
-  [
-    'schema:accessMode',
-    'schema:accessibilityFeature',
-    'schema:accessibilitySummary',
-  ].filter(meta => epub.metadata[meta] === undefined)
-    .forEach(meta => assertions.withAssertions(newMetadataAssertion(meta)));
-  // Recommended metadata
-  [
-    'schema:accessModeSufficient',
-  ].filter(meta => epub.metadata[meta] === undefined)
-    .forEach(meta => assertions.withAssertions(newMetadataAssertion(meta, 'moderate')));
-}
-
-function checkTitle(assertions, epub) {
-  const title = asString(epub.metadata['dc:title']);
-  if (title === '') {
-    assertions.withAssertions(newViolation({
-      title: 'epub-title',
-      testDesc: 'Ensures the EPUB has a title',
-      resDesc: 'Add a \'dc:title\' metadata property to the Package Document',
-      kbPath: '',
-      kbTitle: 'EPUB Title',
-    }));
-  }
-}
-
-function checkPageSource(assertion, epub) {
-  if (epub.navDoc.hasPageList
-    && (epub.metadata['dc:source'] === undefined
-    || epub.metadata['dc:source'].toString() === '')) {
-    assertion.withAssertions(newViolation({
-      title: 'epub-pagesource',
-      testDesc: 'Ensures the source of page breaks is identified',
-      resDesc: 'Add a \'dc:source\' metadata property to the Package Document',
-      kbPath: 'docs/navigation/pagelist.html',
-      kbTitle: 'Page Navigation',
-    }));
-  }
-}
-
 function check(epub, report) {
   winston.info('Checking package...');
-  const assertion = new builders.AssertionBuilder()
-    .withSubAssertions()
-    .withTestSubject(epub.packageDoc.src, asString(epub.metadata['dc:title']));
 
-  // Check a11y metadata
-  checkMetadata(assertion, epub);
+  // normalize title property for future processing
+  // TODO: Can asString() replaced by (epub.metadata['dc:title'] || '').toString().trim()
+  // TODO: or it is possible that metadata['dc:title'] include more than one element?
```

Comment:
yes, this was intended for robustness as there can be multiple titles (although only the first is relevant, as far as I remember).